### DISCUSSION TITLE
Unable to get instrumentation working locally

### DISCUSSION THREAD

**Discussion by Gawdfrey** [vercel/next.js#61290] [Help]
### Summary

I am unable to get instrumentation to work and I've tried countless guides so far..
For the latest attempt I have used the [official template](https://github.com/vercel/next.js/tree/canary/examples/with-opentelemetry) and spun up the collector backend using [this](https://github.com/vercel/opentelemetry-collector-dev-setup).

After spinning the application up and checking out the jaeger ui, I do not see my service registered at all.
I feel like there is some missing piece of information missing from the [official documentation](https://nextjs.org/docs/app/building-your-application/optimizing/open-telemetry), as they claim it should be as easy as this 😅 Is there some environment variables I am missing?

### Example

https://github.com/Gawdfrey/nextjs-instrumentation-test

**Comment by tansanDOTeth** [comment]
Same for me
